lib/Bio/EnsEMBL/DataCheck/Checks/CompareOntologyTerm.pm

```diff
@@ -31,7 +31,7 @@ use constant {
   NAME           => 'CompareOntologyTerm',
   DESCRIPTION    => 'Compare Term counts between current and previous ontology database',
   GROUPS         => ['ontologies'],
-  DATACHECK_TYPE => 'advisory',
+  DATACHECK_TYPE => 'critical',
   DB_TYPES       => ['ontology'],
   TABLES         => ['term']
```

Comment:
```suggestion
  TABLES         => ['ontology', 'term']
```
Since we're now joining to the ontology table, want changes there to trigger re-runs of the datacheck.